subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationResult.java

```diff
@@ -324,11 +375,15 @@ public Object getCustomOperationTraceSerializableModel() {
             class Property {
```

Comment:
This type doesn't have to be stable at all. I suggest just having a single `attributes` property instead of destructuring.